frontends/tests/e2e/base.test.js

```diff
@@ -0,0 +1,42 @@
+import { remote } from "webdriverio";
+import { expect } from "chai";
+
+// --- Test ---
+describe("BitBoxApp Base Test", function () {
+  this.timeout(180000);
+
+  let driver;
+  before(async () => {
+
+    const opts =  {
+      path: '/',
+      port: 4723,
+      capabilities: {
+        platformName: 'Android',
+        'appium:deviceName': 'Android Emulator',
+        'appium:automationName': 'UiAutomator2',
+        'appium:app': `./apk/app-debug.apk`,
+        'appium:noReset': true,
+      }
+    };
+
+    driver = await remote(opts);
+
+    // Switch to WebView if present
+    const contexts = await driver.getContexts();
+    const webview = contexts.find((c) => c.startsWith("WEBVIEW_"));
+    if (webview) await driver.switchContext(webview);
+  });
+
+  after(async () => {
+    if (driver) await driver.deleteSession();
+  });
+
+  it("App main page loads", async () => {
```

Comment:
Can or should the Android tests reuse Playwright tests? The code running on both platforms is nearly identical after all. Or is the plan to write most tests in Playwright and only stuff that is different on Android here?

What is the overall plan? Maybe worth writitng it down somewhere so people who will write tests know how to proceed.

Reply:
@thisconnect is right. As far as I know there isn't really a way to reuse the tests. I will investigating whether we can at least translate one into the other easily or if we should keep it separated. I think the plan is still a bit in-flight and not set in stone yet. I'll make sure to track it in an issue.